packages/fx-core/resource/package.nls.json

```diff
@@ -455,7 +455,7 @@
   "error.debugHandler.InvalidExistingBotArgsError": "botId and botPassword need to be set together if you use existing bot.",
   "error.debugHandler.DebugArgumentEmptyError": "Debug argument '%s' is empty.",
   "error.debugHandler.InvalidTabBaseUrlError": "baseUrl must be a valid https url.",
-  "error.m365.M365TitleNotAcquiredError": "Microsoft 365 title is not acquired yet, please use m365Title/acquire action to acquire Microsoft 365 title.",
+  "error.m365.NotExtendedToM365Error": "Teams app is not extended across Microsoft 365 yet, please use teamsApp/extendToM365 action to extend your Teams app across Microsoft 365.",
```

Comment:
Unable to extend Teams app to Microsoft 365. Use 'teamsApp/extendToM365' action to extend your Teams app to Microsoft 365.